src/wallet/wallet.ts

```diff
@@ -135,6 +145,39 @@ class HathorWalletServiceWallet extends EventEmitter implements IHathorWallet {
    **/
   clearSensitiveData() {
     this.seed = null;
+    this.authXpriv = null;
+  }
+
+  /**
+   * Get auth xpubkey in the 280' purpose path from seed
+   *
+   * @param {String} seed 24 words
+   * @param {Object} options Options with passphrase and networkName
+   *
+   * @return {String} auth xpubkey
+   * @memberof HathorWalletServiceWallet
+   * @inner
+   */
+  static getAuthXPubKeyFromSeed(seed: string, options: { passphrase?: string, networkName?: string } = {}): string {
+    const methodOptions = Object.assign({passphrase: '', networkName: 'mainnet', accountDerivationIndex: '0\''}, options);
+
+    const xpriv = walletUtils.getXPrivKeyFromSeed(seed, methodOptions);
+    const privkey = HathorWalletServiceWallet.deriveAuthXpriv(xpriv);
+
+    return privkey.xpubkey;
+  }
+
+  /**
+   * Derive xpriv from root to the auth specific purpose derivation path
+   *
+   * @param {string} accountDerivationIndex String with derivation index of account (can be hardened)
```

Comment:
What's this parameter? As far as I understood this is not the `xpriv` param, right?

Reply:
Fixed in 35809f7e9114b604a02e97799a11921286c42ee2